pkg/docker/docker.go

```diff
@@ -18,41 +18,91 @@ import (
 	docker "github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/archive"
 	"github.com/kyma-project/cli/internal/minikube"
+	"github.com/kyma-project/cli/pkg/step"
 )
 
 const (
 	defaultRegistry = "index.docker.io"
 )
 
-// DockerErrorMessage is used to parse error messages coming from Docker
-type DockerErrorMessage struct {
+type dockerClient struct {
+	*docker.Client
+	// DockerClientService
+}
+
+type kymaDockerClient struct {
+	kymaDocker ClientService
```

Comment:
This is the real docker client, not the Kyma client. This makes it all very confusing and misleading.

Reply:
fixed.